Update Readme for LayoutLib publish (#1320)
cashapp/paparazzi: libs/README.md

   layoutlib = "2022.2.1-5128371"
   layoutlibPrebuiltSha = "5128371"
   ```

> 5. Create branch using name `**/publish-layoutlib**` and push to Github

Reviewer: ```suggestion
5. Create a branch that matches the `**/publish-layoutlib**` filter and push to GitHub.
```

> 6. In [Github Actions][publish_action], run `publish-layoutlib` action using your branch name you created.

Reviewer: ```suggestion
6. Go to [GitHub Actions][publish_action] and run the `publish-layoutlib` action using this branch.
```

> [nexus]: https://oss.sonatype.org/

Reviewer: ```suggestion
```
